Sources/Hyperwallet.swift

```diff
@@ -907,6 +907,29 @@ public final class Hyperwallet: NSObject {
                                        completionHandler: transferMethodConfigurationFieldResponseHandler(completion))
     }
 
+    /// Returns the transfer method update configuration field set for the User that is associated with the
+    /// authentication
```

Comment:
Combine this and the next line - no need to have an extra new line here

Reply:
Let me check it.